### DISCUSSION TITLE
My Quirky Set-Up

### DISCUSSION THREAD

**Discussion by Jimmbo** [rxhanson/RectanglePro-Community#631] [Show and tell]
Not everyone wants to use Rectangle this way, but if it's your speed, this setup works great. The keyboard shortcuts are a big part of the appeal (I use control key as the modifier).

This is really for individual placement, not stacking. When I have complicated stacking needs, the presets are easily chosen. This scheme lets me effortlessly place and size 95% of windows exactly like I prefer. Used in tandem with [Hazeover](https://hazeover.com) (which darkens everything but the front window), it's particularly useful.

Top image is perfect for word processing windows
Middle image is perfect for Finder windows, productivity app windows, etc.
Bottom image is perfect for browser (and email client) windows

_Note: if you're running out of keyboard shortcut real estate, a keyboard with number pad is one answer. The number and symbol keys there are distinct from the same ones in the main keyboard. This opens up dozens/hundreds more shortcut options...while also making number entry a pleasure._

![1-3](https://github.com/user-attachments/assets/b1c267be-8733-4eef-869a-f3b51bf5192c)
![4-6](https://github.com/user-attachments/assets/7b070c7e-7e88-4994-a320-533eac50cdfd)
![7-9](https://github.com/user-attachments/assets/8a9b56c4-2d41-48f1-9903-5ff75e1252d2)

**Comment by rxhanson** [comment]
Nice, thanks for sharing!
